libs/wingsdk/src/target-tf-aws/platform.ts

```diff
@@ -1,3 +1,9 @@
+import {
+  SecretsManagerClient,
+  GetSecretValueCommand,
+  CreateSecretCommand,
+  UpdateSecretCommand,
+} from "@aws-sdk/client-secrets-manager";
```

Comment:
Could this be changed to a dynamic import that is only loaded if there are secrets?
This platform will be loaded for all tf-aws apps and unfortunately AWS SDKs are expensive. The benchmarks are showing an flat increase of almost ~100ms which is not negligible.

Reply:
Yea Im pretty sure we can use dynamic imports. Thank goodness for the benchmarks @MarkMcCulloh because I ALWAYS overlook using dynamic imports 😁